src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -127,25 +132,29 @@ function compute_sgs_flux!(
     # apply surface BC as SGS flux at lowest level
     lg_surf = CC.Fields.local_geometry_field(axes(ρ_f))[kf_surf]
     sgs_flux_h_tot[kf_surf] =
-        CCG.Covariant3Vector(CCG.WVector(get_ρe_tot_flux(surf)), lg_surf)
+        CCG.Covariant3Vector(surface_conditions.ρ_flux_h_tot, lg_surf)
     sgs_flux_q_tot[kf_surf] =
-        CCG.Covariant3Vector(CCG.WVector(get_ρq_tot_flux(surf)), lg_surf)
-    ρu_flux = edmf.zero_uv_fluxes ? FT(0) : get_ρu_flux(surf)
-    ρv_flux = edmf.zero_uv_fluxes ? FT(0) : get_ρv_flux(surf)
+        CCG.Covariant3Vector(surface_conditions.ρ_flux_q_tot, lg_surf)
+    ρ_flux_uₕ = if edmf.zero_uv_fluxes
+        CC.Geometry.AxisTensor(
```

Comment:
This can be simplified to `zero(ρ_flux_uₕ)`. Same for the other places where it appears.